src/Autodesk.Forge.DesignAutomation/ApiClientFactory.cs

```diff
@@ -35,8 +35,9 @@ public DesignAutomationClient CreateClient(string agent = null)
         {
             var httpClientFactory = services.GetRequiredService<IHttpClientFactory>();
             var typedClientFactory = services.GetRequiredService<ITypedHttpClientFactory<ForgeService>>();
-            var forgeService = typedClientFactory.CreateClient(httpClientFactory.CreateClient(agent));
-            var client = ActivatorUtilities.CreateInstance<DesignAutomationClient>(
+            var client = httpClientFactory.CreateClient(agent?? ForgeAgentHandler.defaultAgentName);
```

Comment:
`agent = null` won't work because httpClientFactory.CreateClient(agent) throws exception for null input.
Instead, when I pass agent = null in DesignAutomationClientFactory.CreateClient, I hope to use the default forge app ClientId/ClientSecret. So we can use default forge app and forge apps in Agents list via DesignAutomationClientFactory easily

The PR for forge-api-dotnet-design.automation: https://github.com/Autodesk-Forge/forge-api-dotnet-design.automation/pull/33